acp/README.md

```
@@ -117,25 +110,11 @@ The name of the required 'registerer' relation is:
-- `expr: ownerMalicious`
-- `expr: owner_new`
-- `expr: reader+owner`
-- `expr: reader-owner`
-- `expr: reader - owner`
-
 Here are some valid expression examples. Assuming these `expr` are under a required permission label:
-- `expr: owner`
-- `expr: owner + reader`
-- `expr: owner +reader`
-- `expr: owner+reader`
+- `expr: ` 
```

markdownlint-cli2 (0.18.1)

Spaces inside code span elements

(MD038, no-space-in-code)